README.md

```diff
@@ -119,6 +119,26 @@ The transport can be configured with the `board_microros_transport = <transport>
     set_microros_native_ethernet_transports(local_mac, local_ip, agent_ip, agent_port);
     ```
 
+  - `custom`
```

Comment:
We should move this to this section:
https://github.com/micro-ROS/micro_ros_platformio/blob/main/README.md#custom-transport

Reply:
Maybe we should rename this section as how to extend available library transports because concepts are similar but not the same